extensions/ql-vscode/src/variant-analysis/code-scanning-pack.ts

```diff
@@ -0,0 +1,77 @@
+import { join } from "path";
+import type { App } from "../common/app";
+import type { QueryLanguage } from "../common/query-language";
+import type { CodeQLCliServer } from "../codeql-cli/cli";
+import type { QlPackDetails } from "./ql-pack-details";
+import { getQlPackFilePath } from "../common/ql";
+
+export async function getCodeScanningPack(
```

Comment:
Perhaps something like `resolveCodeScanningQueryPack` would be a slightly better name?